api/test/robot_tests/cmd/cmd.robot

```diff
@@ -108,7 +108,7 @@ POST /cmd/executeFunction with invalid function name
     ${response}=    POST
     ...    ${API_URL}/cmd/executeFunction
     ...    json=${data}
-    ...    expected_status=${STATUS_INTERNAL_SERVER_ERROR}
+    ...    expected_status=${STATUS_BAD_REQUEST}
```

Comment:
Why updating this expected status ? 
It looks like it's still an internal server error